exporter/views.py

```diff
@@ -34,14 +36,88 @@ def get_original_asset_id(download_url):
     that identifies this image uniquely on loc.gov
     """
     if download_url.startswith("http://tile.loc.gov/"):
-        pattern = r"/service:([A-Za-z0-9:\-]*)/"
+        pattern = r"/service:([A-Za-z0-9:\-]+)/"
         asset_id = re.search(pattern, download_url)
-        assert asset_id
-        return asset_id.group(1)
+        if not asset_id:
+            logger.error(
+                "Couldn't find a matching asset ID in download URL %s", download_url
+            )
+            raise AssertionError
```

Comment:
This could be some other exception class but I care less about that than that it’s checked since it seems like this condition will usually be telling us that we got some bogus data from the API, and that edge case probably means we need to get some records fixed